tests/integration/annotation_import/test_export_v2_classifications.py

```diff
@@ -0,0 +1,192 @@
+import time
+import labelbox as lb
+from labelbox.data.annotation_types.data.video import VideoData
+import labelbox.types as lb_types
+from labelbox.schema.annotation_import import AnnotationImportState
+
+
+def test_export_v2_video(client, configured_project_without_data_rows,
```

Comment:
Nit: This test feels like it doesn't belong inside `annotation_import`. It may be fit better inside `test_export.py` or you could even create a new folder `exports` and put all exports related tests there

Reply:
yes I was thinking we should separate exports.... the problem is, we already have exports v2 tests as a part of many annotation export test files, like see this tests/integration/annotation_import/test_model_run.py as an example. I can move just this one test for now

Reply:
@kkim-labelbox I have looked at this and even moving 1 file in a separate folder requires refactoring or duplicating conftest... I think our conftest would benefit from refactoring any way, but I propose we should try to do it as we go or later, once the exporter v2 goes live
I think I could start doing small incremental changes every time, but doing as a part of this PR will distract me from finishing all v2 work